Directory.Packages.props

```diff
@@ -5,6 +5,7 @@
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
 
     <MicroBuildVersion>2.0.107</MicroBuildVersion>
+    <VisualStudioThreadingVersion>17.6.35-alpha-gd646d3a5e1</VisualStudioThreadingVersion>
```

Comment:
This will be replaced with an official build when https://github.com/microsoft/vs-threading/pull/1159 merges.